src/mongo-db/blog.schema.ts

```diff
@@ -0,0 +1,19 @@
+import { Schema, Prop, SchemaFactory } from '@nestjs/mongoose';
+import mongoose from 'mongoose';
+import { Post } from './posts.schema';
+import { User } from './user.schema';
+import { Content } from './content.schema';
+
+@Schema()
+export class Blog {
+  @Prop({ type: mongoose.Schema.Types.ObjectId, ref: 'Posts' })
+  posts: Post[];
+
+  @Prop({ type: mongoose.Schema.Types.ObjectId, ref: 'Users' })
+  users: User[];
+
+  @Prop({ type: mongoose.Schema.Types.ObjectId, ref: 'Users' })
```

Comment:
This ref could be removed, let`s consider that content is referenced type, thinking this way:

<img width="628" alt="image" src="https://github.com/natalielie/admin-dashboard/assets/62031924/78cc1583-b034-4cab-aed1-ffb87a2e99ab">

Considering the mentioned scheme, let`s update the fields